sale_order_variant_mgmt/wizard/sale_manage_variant_view.xml

```diff
@@ -18,10 +18,10 @@
                        field_x_axis="value_x"
                        field_y_axis="value_y"
                        field_value = "product_uom_qty"
-                       field_att_disabled="disabled"
                        x_axis_clickable="0"
                        y_axis_clickable="0"
                 />
+                       <!-- field_att_disabled="disabled" TODO: gives error -->
```

Comment:
:thinking: I don't understand this...

Reply:
Ah, this is a remainder of all system for disabling "empty" cells. In first version of the widget, I use extra fields for setting HTML attributes (in this case, field `disabled` sets HTML `disabled` attribute), ala QWeb t-att directive. Now it's not needed as missing cells are covered, although this was a powerful option for modifying widget aspect. I'm removing this comment.